docs/feature_rgb_matrix.md

```diff
@@ -281,8 +281,10 @@ You can disable a single effect by defining `DISABLE_[EFFECT_NAME]` in your `con
 
 |Define                                                 |Description                                    |
 |-------------------------------------------------------|-----------------------------------------------|
+|`#define DISABLE_RGB_MATRIX_SOLID_COLOR`               |Disables `RGB_MATRIX_ALPHAS_MODS`              |
```

Comment:
```suggestion
|`#define DISABLE_RGB_MATRIX_SOLID_COLOR`               |Disables `RGB_MATRIX_SOLID_COLOR`              |
```
Didn't know this could be done. Might present a problem if someone were to define all of these - there should be *at least* one animation available... 🤔